src/styles/navbar.module.scss

```
@@ -0,0 +1,213 @@
+
+
+  .logo{
+      width: 50px;
+      height:50px;
+    }
```

```suggestion
    }
    
```
let's add space between selectors